src/utils.ts

```diff
@@ -586,3 +586,7 @@ export const filterObjectKeys = (
     .reduce((cur, key) => {
       return Object.assign(cur, { [key]: obj[key] });
     }, {});
+
+export const isKillSwitchOn = () => !!process.env['LUMIGO_SWITCH_OFF'];
```

Comment:
please verify that this is correct, if i recall correctly this should be a string comparison with very specific cases where it's "true"

Reply:
used already existing function isSwitchedOff() instead